collections/sequence.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 // DefaultSequenceStart defines the default starting number of a sequence.
-const DefaultSequenceStart uint64 = 1
+const DefaultSequenceStart uint64 = 0
```

Comment:
modified this to make it match the account number semantics, which start from zero.